python/persistenceClustering1.py

```diff
@@ -0,0 +1,124 @@
+threshold12.ThresholdRange = [-1.0, -0.1]
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKPersistenceDiagram1)
+threshold1.Scalars = ["CELLS", "PairIdentifier"]
+threshold1.ThresholdRange = [-0.1, 999.0]
+
+# create a new 'Extract Surface'
+extractSurface7 = ExtractSurface(Input=threshold12)
+
+# create a new 'Tube'
+tube7 = Tube(Input=extractSurface7)
+tube7.Scalars = ["POINTS", "CriticalType"]
+tube7.Vectors = ["POINTS", "Coordinates"]
+tube7.Radius = 0.75
+
+# create a new 'Threshold'
+persistenceThreshold0 = Threshold(Input=threshold1)
+persistenceThreshold0.Scalars = ["CELLS", "Persistence"]
+persistenceThreshold0.ThresholdRange = [10.0, 9999.0]
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints11 = TTKIcospheresFromPoints(Input=persistenceThreshold0)
+
+# create a new 'Extract Surface'
+extractSurface6 = ExtractSurface(Input=persistenceThreshold0)
+
+# create a new 'Tube'
+tube6 = Tube(Input=extractSurface6)
+tube6.Scalars = ["POINTS", "CriticalType"]
+tube6.Vectors = ["POINTS", "Coordinates"]
+tube6.Radius = 0.75
+
+# create a new 'TTK TopologicalSimplification'
+tTKTopologicalSimplification1 = TTKTopologicalSimplification(
+    Domain=slice1, Constraints=persistenceThreshold0
+)
+tTKTopologicalSimplification1.ScalarField = ["POINTS", "SplatterValues"]
+tTKTopologicalSimplification1.InputOffsetField = [None, ""]
+tTKTopologicalSimplification1.VertexIdentifierField = [None, ""]
+
+# create a new 'TTK MorseSmaleComplex'
+tTKMorseSmaleComplex1 = TTKMorseSmaleComplex(Input=tTKTopologicalSimplification1)
+tTKMorseSmaleComplex1.ScalarField = ["POINTS", "SplatterValues"]
+tTKMorseSmaleComplex1.OffsetField = ["POINTS", "SplatterValues"]
+
+
+# create a new 'Resample With Dataset'
+resampleWithDataset1 = ResampleWithDataset(
+    SourceDataArrays=OutputPort(tTKMorseSmaleComplex1, 3),
+    DestinationMesh=tableToPoints1,
+)
+resampleWithDataset1.CellLocator = "Static Cell Locator"
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints1 = TTKIcospheresFromPoints(Input=resampleWithDataset1)
+tTKIcospheresFromPoints1.Radius = 0.05
+
+# create a new 'Threshold'
+threshold2 = Threshold(Input=OutputPort(tTKMorseSmaleComplex1, 1))
+threshold2.Scalars = ["CELLS", "SeparatrixType"]
+
+# create a new 'TTK GeometrySmoother'
+tTKGeometrySmoother1 = TTKGeometrySmoother(Input=threshold2)
+tTKGeometrySmoother1.IterationNumber = 10
+tTKGeometrySmoother1.InputMaskField = [None, ""]
+
+# create a new 'Extract Surface'
+extractSurface1 = ExtractSurface(Input=tTKGeometrySmoother1)
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints3 = TTKIcospheresFromPoints(Input=tTKMorseSmaleComplex1)
+tTKIcospheresFromPoints3.Radius = 0.075
+
+# create a new 'Tube'
+tube1 = Tube(Input=extractSurface1)
+tube1.Scalars = ["POINTS", "CellDimension"]
+tube1.Vectors = [None, ""]
+tube1.Radius = 0.025
+
+# save the output(s)
+SaveData('PersistenceDiagram.vtu', tTKPersistenceDiagram1)
```

Comment:
for this example, what you want to output is the raw output points with their classification (`resampleWithDataset1`).
then store that in `csv` file format as this is more convenient to manipulate in batch mode post-processing.
also remove the generation of any graphical primitive (spheres, tubes, etc.)

the idea is to make the script as **short** as possible :)
I have updated the contributor instructions accordingly:
https://github.com/topology-tool-kit/ttk-data/blob/dev/CONTRIBUTING.md